### ISSUE TITLE
same command on osx and linux has different output

### ISSUE THREAD

**Issue by OAlienO** [bwall/HashPump#19]
On Ubuntu 16.04
![linux](https://user-images.githubusercontent.com/18167730/37915160-6ec8648a-314b-11e8-8f62-c7d70314385a.png)
On OSX 10.12.6
![osx](https://user-images.githubusercontent.com/18167730/37915164-715c0058-314b-11e8-9dfb-d680e3308533.png)
hashpump version I used in both system is `Version 1.2.0`
```
hashpump -s '9863bb3cecccfdb82f689e2ddbdcd9c7ea3c069eb73a730d149c3aae2d60b7c0' --data 'user=someone' -a 'user=admin' -k 44
```
The command is used to solve this CTF challenge [BambooFox CTF 2018 - baby-lea](https://bamboofox.cs.nctu.edu.tw/courses/5/challenges/93)
It's **SHA256 Length Extension Attack**
hashpump seems to output a wrong answer on OSX 10.12.6

**Comment by OAlienO** [comment]
Oh, I got it XD
The issue here #16 related to **SHA256 padding bug** is fixed after the release of v1.20
And Ubuntu user build the binary from source
On the other hand, OSX user get the binary from `brew install hashpump` which still has the bug
For OSX user, build the binary from source will solve the problem

Here is my fix : 
Follow [this guide](https://www.anintegratedworld.com/mac-osx-fatal-error-opensslsha-h-file-not-found/) to install openssl and link the library
```
brew install openssl
cd /usr/local/include 
ln -s ../opt/openssl/include/openssl .
```
`git clone` the repository and `make`
```
git clone https://github.com/bwall/HashPump.git
cd HashPump
make
```
Then the generated binary in the HashPump folder will work perfectly

**Comment by OAlienO** [comment]
Since the problem is solved, I will close the issue
btw, it looks like I am talking to myself XD
